src/Ojs/JournalBundle/Listeners/JournalAnnouncementMailer.php

```diff
@@ -26,6 +26,7 @@ public static function getSubscribedEvents()
     public function onJournalAnnouncementPostCreate(JournalItemEvent $event)
     {
         $this->sendAnnouncementMail($event, JournalAnnouncementEvents::POST_CREATE);
+        $this->sendAnnouncementMailToSubscribers($event, JournalAnnouncementEvents::POST_CREATE);
```

Comment:
There won't be any need for this once you change `sendAnnouncementMail` method as I described in one of the comments.